fili-core/src/main/java/com/yahoo/bard/webservice/application/AbstractBinderFactory.java

```diff
@@ -1098,6 +1102,32 @@ protected DruidWebService buildMetadataDruidWebService(ObjectMapper mapper) {
         return buildDruidWebService(DruidClientConfigHelper.getMetadataServiceConfig(), mapper);
     }
 
+    /**
+     * Create a ResponseFormatResolver for Servlet objects.
+     * <p>
+     * Currently default types are json, jsonapi and csv types.
+     *
+     * @return A ResponseFormatResolver
+     */
+    protected ResponseFormatResolver buildResponseFormatResolver() {
+        return (format, containerRequestContext) -> {
+            Map<String, String> formatsMap = new LinkedHashMap<>();
+            formatsMap.put("application/json", "json");
+            formatsMap.put("application/vnd.api+json", "jsonapi");
+            formatsMap.put("text/csv", "csv");
+
+            String headerFormat = containerRequestContext.getHeaderString("Accept");
+            if (format != null || headerFormat == null) {
+                return format;
+            }
+            return formatsMap.entrySet().stream()
```

Comment:
Why stream if we're just doing a map lookup?

Reply:
It's not a map lookup. A valid header format could be any string that contains our map keys. For example:
```
Accept : text/html, text/csv
```